mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -155,6 +163,16 @@ public static void setActivityTheme(@NonNull final Activity activity, String the
             themeRes = R.style.HABDroid_Light;
         }
         activity.setTheme(themeRes);
+
+        if (VERSION.SDK_INT >= VERSION_CODES.LOLLIPOP) {
+            TypedValue typedValue = new TypedValue();
+            activity.getTheme().resolveAttribute(R.attr.colorPrimary, typedValue, true);
+            activity.setTaskDescription(
+                    new ActivityManager.TaskDescription(activity.getString(R.string.app_name),
+                            BitmapFactory.decodeResource(activity.getResources(),
+                                    R.mipmap.icon_round),
+                            typedValue.data));
```

Comment:
I realize that's total nitpicking, but wouldn't make this more sense?

```
activity.setTaskDescription(new ActivityManager.TaskDescription(
        activity.getString(...),
        BitmapFactory.decodeResource(...),
        typedValue.data));
```

Reply:
You're right, that looks better.

> that's total nitpicking

Doesn't care as long code gets better :)